### PR TITLE
Add deepin 23 repositories

### PR DESCRIPTION
**Pull request by wineee**
- rename old deepin to `deepin 20`
- use deepin instead of Deepin (when using deepin as a standalone branding name, the first letter needs to be in lower-case)
- add  repositories(main, community, commercial) for `deepin 23`

### CODE REVIEW

**Comment by AMDmi3** [comment]
New repository submissions are required to provide links to package sources.

**Comment by wineee** [comment]
`commercial` (non-free) packages can not provide package sources,  how to exclude it

**Review by Volato999 — APPROVED** [review]

**Comment by UTsweetyfish** [comment]
Any progress for this? @AMDmi3

**Comment by AMDmi3** [comment]
Hopefully I'll get to it in January. This requires testing on staging repology instance I'm not able to run right now.

**Comment by UTsweetyfish** [comment]
> Hopefully I'll get to it in January. This requires testing on staging repology instance I'm not able to run right now.

kindly ping for review :)

File: repos.d/deb/deepin.yaml [hunk 1]
```diff
@@ -26,3 +26,30 @@
...
   groups: [ all, production, deepin ]
+
+- name: deepin_23
+  type: repository
+  desc: deepin 23
+  statsgroup: Debian+derivs
+  family: debuntu
+  ruleset: [ debuntu, deepin ]
+  color: '949393'
+  minpackages: 28000
```
Comment (AMDmi3) [inline repos.d/deb/deepin.yaml:37]: It only happens to have 9009 packages (is this expected?), minpackages needs to be fixed to 8000 or so.

**Review by AMDmi3 — CHANGES_REQUESTED** [review]

**Comment by AMDmi3** [comment]
> `commercial` (non-free) packages can not provide package sources, how to exclude it

Please provide examples. Don't these still have package recipes at least?

File: repos.d/deb/deepin.yaml (hunk 1 above)
Comment (wineee) [reply repos.d/deb/deepin.yaml]: Fixed to 8000 now
cc @UTsweetyfish is this expected to have 9009 packages?

**Comment by wineee** [comment]
>Please provide examples.

eg. [libnvidia-encode1](https://community-packages.deepin.com/deepin/beige/pool/commercial/n/nvidia-graphics-drivers/libnvidia-encode1_530.41.03-3deepin2_arm64.deb)

>Don't these still have package recipes at least?

deepin is not officially provided any package recipes website

File: repos.d/deb/deepin.yaml (hunk 1 above)
Comment (AMDmi3) [reply repos.d/deb/deepin.yaml]: \* I'm asking because deepin 20 had much more (29k) packages.

Comment (wineee) [reply repos.d/deb/deepin.yaml]: deepin 20 is based on debain, but deepin 23 is an independent distribution. So this change is possible

**Review by AMDmi3 — APPROVED** [review]

**Pull request merged**